### ISSUE TITLE
Cannot train LORA anymore - No settings changes - 0.1+ loss

### ISSUE THREAD

**Issue by Evil-Dragon** [bmaltais/kohya_ss#1098]
Started up a training session today to train a new LORA, every setting was the same as previous LORA's i've trained and i've made no changes to the PC i'm using. 

Suddenly every LORA that I try to train ends up going from low loss to 0.1+ loss after 100 steps. I've even tried setting up a simple LORA training session with 1 picture of a ball and basic training settings and it still bombs out completely in sample images and 0.1+ loss rate.

Even my existing training sessions will blow up and they were successfully trained with zero changes to training parameters.

~~I've tried rolling back to a few older releases, i've tried new latest graphics drivers, i've tried both versions of torch (1 and 2) and done as much as I can to test my VRAM but i'm at a complete loss. I've checked to make sure Windows isn't corrupted.~~

~~I've trained loads of LORA's before and never had any issue until now so i don't know what to do next. Could my Asus RTX 3080 12GB be dying? I've only had it just over 1 year and a few months.~~

~~Any suggestions are welcome.~~

**EDIT:** I've been able to get kohya_ss-21.7.7 release to train but only on Torch 1. Torch 2 threw an error about Pytorch: Torch 2.0.1+cu118 17:22:50-122539 ERROR Could not load torch: cuDNN version incompatibility: PyTorch was compiled against (8, 7, 0) but found runtime version (8, 6, 0). PyTorch already comes bundled with cuDNN. One option
to resolving this error is to ensure PyTorch can find the bundled cuDNN.one possibility is that there is a conflicting cuDNN in LD_LIBRARY_PATH.

Is Torch 2 broken some how? Bug?

**Comment by bmaltais** [comment]
Try deleting the venv and re-run setup... Might help.

**Comment by Evil-Dragon** [comment]
I've tried that several times on the latest release and only version i've got training at the moment with success and not blowing up is v21.7.7 on Torch 1. I've been unable to train on the latest releases since yesterday.

**Comment by Evil-Dragon** [comment]
Update: So I can confirm Torch 1 version works on the latest version: v21.7.16 but Torch 2 is a no-go at all and still blows up training. Not sure why this is?

**Comment by bmaltais** [comment]
Hard to tell... But most probably a driver issue. Apparently there is a new Nvidia driver that is breaking torch 2... Perhaps that got installed on your windows computer?

**Comment by Evil-Dragon** [comment]
I've tried rolling back to Nvidia drivers at least 2 releases using Nvclean, to at least the 30th May drivers and still Torch 2 will not train and blows up. Torch 1 will train fine without issues on the latest kohyss version.

Just really stuck with what has gone wrong with Torch 2 all of a sudden.

**Comment by dimtoneff** [comment]
the cudnn version in the zip file here in the repo is 8.6... this blows because it wants the 8.7. If you find and replace the 8.6 library files with 8.7... it might work

**Comment by Evil-Dragon** [comment]
I think I might have just found that out, because I just downloaded the standalone (non-git pulled) version and just ran it without even copying the CUDNN files and now it's training with Torch 2 without blowing up (so far after 700 steps)

**Comment by bmaltais** [comment]
This is interesting ... The culprit would be the cudaNN DLL hack then... Perhaps I should just remove this option from future releases to prevent this annoying issue... Will investigate this...

**Comment by Robo0890** [comment]
I have also had some similar issues. 
My lora and full dreambooth runs all started out fine at 0.07 loss, then jump to 0.14, 0.17, .2, and finally **nan**.
The sample images resulting from the training before nan are all messed up. 

I thought it was from updating the gui, so I reverted to older versions. But still no dice. I just got fed up and switched to using the Automatic 1111 extension, but that method took about 2 hrs to train for 6 epochs. MENTAL.

But hopefully disabling the cudaNN DLL does the trick.
